CHANGES.md

```diff
@@ -1,15 +1,20 @@
-# next -- TBA
+# Version 1.2 -- 2024-08-30
 
 ## New Features
 
 * Add `mir_str_slice_value` and `mir_str_slice_range_value` functions, which
   allow taking `&str` slices. For more information, see the documentation in the
   [SAW manual](https://github.com/GaloisInc/saw-script/blob/master/doc/manual/manual.md#string-slices).
 
+* Add `mir_mux_values` command for muxing two MIR values.
+
+* Error messages have been improved such that the locations printed with type errors and other diagnostics now have a much stronger connection with reality.
+
+* Add support for GHC 9.8.
+
 ## Bug fixes
 
-* The locations printed with type errors (in particular) and other diagnostics
-  now have a much stronger connection with reality.
+* Fix bug that caused MIR to override incorrectly rejected when applied to multiple const slice arguments.
```

Comment:
ENOPARSE; guess you meant "to incorrectly reject overrides with multiple const slice arguments".